assets/scss/bulmaswatch/_variables.scss

```diff
@@ -0,0 +1,56 @@
+////////////////////////////////////////////////
+// PULSE
+////////////////////////////////////////////////
+$orange:       #FF7518;
+$yellow:       #f1c40f;
+$green:        #2ecc71;
+$turquoise:    #14a789;
+$blue:         #3498db;
+$purple:       #8e44ad;
+$red:          #e74c3c;
+
+$primary: $purple !default;
+$warning: $orange;
+
+$orange-invert: #fff;
+$warning-invert: $orange-invert;
+
+$background: lighten($primary, 50);
+
+$family-sans-serif: 'Muli', "Helvetica Neue", Arial, sans-serif;
+$body-size: 14px;
+
+$subtitle-color: darken($primary, 10);
+
+$radius: 0;
+$radius-small: 0;
+$radius-large: 0;
+
+$link: $turquoise;
+$link-hover: lighten($link, 5);
+$link-focus: darken($link, 10);
+$link-active: darken($link, 10);
+
+$button-hover-color: lighten($primary, 10);
+$button-hover-border-color: lighten($primary, 10);
+
+$button-focus-color: darken($primary, 10);
+$button-focus-border-color: darken($primary, 10);
+$button-focus-box-shadow-size: 0 0 0 0.125em;
+$button-focus-box-shadow-color: rgba($primary, 0.25);
+
+$button-active-color: darken($primary, 10);
+$button-active-border-color: darken($primary, 10);
+
+$navbar-background-color: $primary;
+$navbar-item-color: #fff;
+$navbar-item-hover-color: $navbar-item-color;
+$navbar-item-active-color: $navbar-item-color;
+$navbar-item-hover-background-color: rgba(#000, 0.2);
+$navbar-item-active-background-color: rgba(#000, 0.2);
```

Comment:
`0.2` should be written without a leading zero as `.2`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#leadingzero)